src/utils.js

```diff
@@ -131,6 +132,8 @@ function computeDefaults(schema, parentDefaults, definitions={}) {
     // Use referenced schema defaults for this node.
     const refSchema = findSchemaDefinition(schema.$ref, definitions);
     defaults = computeDefaults(refSchema, defaults, definitions);
+  } else if (isFixedItems(schema)) {
+    defaults = schema.items.map(itemSchema => computeDefaults(itemSchema, undefined, definitions));
```

Comment:
This needs a test to be added in the `getDefaultFormState` test suite in `test/utils_test.js` (`getDefaultFormState` uses this function).